operations/_objects.md

```diff
@@ -182,6 +182,15 @@ A [Dictionary](#dictionary) object where the keys are [Language](languages.md#la
 | `NetValue` | number | required | The net value that the tax is calculated from. |
 | `TaxValue` | number | required | The value of the tax. |
 
+### Hybrid identifier
+
+A hybrid identifier is a string with a specific format that allows the use of alternative unique identifiers, in addition to the entity's unique identifier. When using these alternative identifiers, a corresponding prefix must be included as part of the value.
+
+| Identifier | Prefix | Value example |
+| :-- | :-- | :-- | :-- |
+| Primary identifier | - | `a01bc7c3-cfa2-4ad6-a360-5cb8b4004ab5` |
```

Comment:
nitpick: having `-` as Prefix is misleading, consider keeping the field empty or put something like `(none)` there.